elastica/rod/protocol.py

```diff
@@ -6,14 +6,18 @@
 from elastica.systems.protocol import SystemProtocol, SlenderBodyGeometryProtocol
 
 
-class _RodEnergy(Protocol):
+class _CosseratRodStrainEnergy(Protocol):
```

Comment:
Since it includes translational and rotational energy, why not just call it `_CosseratRodEnergy`

Reply:
Yup. [c5319a6](https://github.com/GazzolaLab/PyElastica/pull/405/commits/c5319a6dbc35847af8169dd506ec5ff33ea90f56)